config/armsve/bli_cntx_init_armsve.c

```diff
@@ -33,22 +33,25 @@
 */
 
 #include "blis.h"
-#include <sys/auxv.h>
-
-#ifndef HWCAP_SVE
-#define HWCAP_SVE (1 << 22)
-#endif
 
 void bli_cntx_init_armsve( cntx_t* cntx )
 {
-	if (!(getauxval( AT_HWCAP ) & HWCAP_SVE))
+	// Set default kernel blocksizes and functions.
+	bli_cntx_init_armsve_ref( cntx );
+
+	// If we are autodetecting the correct aarch64 config, then we have to make sure
+	// that SVE instructions are actually available since these are used in determining
+	// the register blocksizes.
+	#ifdef BLIS_FAMILY_ARM64
+	uint32_t family, model, features = 0;
+	bli_cpuid_query( &family, &model, &features );
```

Comment:
I'm going to try to apply this patch, but I'm almost sure it won't work. ARM64 doesn't have cpuid, and the source code for `bli_cpuid_query()` seems to be totally made for x86 only.